### ISSUE TITLE
Communication with web site ( duplicate )

### ISSUE THREAD

**Issue by iWedmak** [feross/webtorrent-desktop#907]
Previos issue https://github.com/feross/webtorrent-desktop/issues/900 - has been closed to early

> is there a way or can it be done in future?
> for example:
> specifically load torrent or magnet though webtorrent?
> get back from webtorrent local link to video file on user machine, so it can be playd through video player on site ?
> so basically it be nice to have a way to communicate with webtorrent from site, so it can be used as service on user machine
________________________________________________________________
> You can already paste magnet links from anywhere into WebTorrent Desktop
> 
> You can also set WebTorrent Desktop as your default magnet link and torrent file handler, after which you can just click on a magnet link and it should open in the app
________________________________________________________________
> I know about this, and this is not my question/suggestion.
> I don't want webtorrent to be default torrent client, its not for that, at this point.
> 
> I need a way of communicate with webtorrent from outside. Pragmatically

**Comment by mathiasvr** [comment]
You don't have to open a new issue.

I think what you need is [WebTorrent](https://github.com/feross/webtorrent/) and not the WebTorrent Desktop app. You can take a look at the docs here https://github.com/feross/webtorrent/tree/master/docs.

**Comment by iWedmak** [comment]
nope, if we speak about user with webtorrent client it will be nice to have ability to know that he have webtorrent client, and minimal api to interact with it,
webtorrent for node will work on server, not user machine, webtorrent desktop works on user machine and have access to user hard and system, so it can be users just as service on user side. what im talking about is have been implemented (patialy and not quet work) in https://github.com/torrentsTime/embed

basically i want ability not to use vlc or webtorrent video player,i want to use my own, in browser, where user is downloads(partially) video file to his device (machine), webtorrent is signals to site (i duno maybe by inserting in html, or changing url of page) web sites get route to local file on user machine and can play it, add to this subtitles and so on, or just add it to play list.

**Comment by dcposch** [comment]
> basically i want ability not to use vlc or webtorrent video player,i want to use my own, in browser, where user is downloads(partially) video file to his device (machine), webtorrent is signals to site (i duno maybe by inserting in html, or changing url of page) web sites get route to local file on user machine and can play it, add to this subtitles and so on, or just add it to play list.

@iWedmak I don't really understand what you're asking for, but it sounds complicated.

Please do not make duplicate issues.

**Comment by iWedmak** [comment]
are You kidding me? This is a criteria? It is too complicated that's all? like you trying to do something simple and easy.
and please reopen issue, cose this is not an answer, this is fiction
